kanidm_unix_int/src/daemon.rs

```diff
@@ -338,10 +331,9 @@ async fn handle_client(
             }
             ClientRequest::Status => {
                 debug!("status check");
-                if cachelayer.test_connection().await {
-                    ClientResponse::Ok
-                } else {
-                    ClientResponse::Error
+                match cachelayer.test_connection().await {
```

Comment:
Will clippy complain if you match on a bool?

Reply:
Yeah I think I over complicated this, I'll revert.